Release 1.0.14-beta6 aka Beta Breakers

nerdneha opened the pull request:
too fast too beta numero 6

Co-Authored-By: Brendan Forster <brendan@github.com>

nerdneha committed:
beta-breakers

too fast too beta - #6

Co-Authored-By: Brendan Forster <brendan@github.com>

shiftkey commented:
@nerdneha I just bumped CircleCI to run again, it couldn't find `yarn` and so the build failed 🤔🤔🤔

shiftkey committed:
should we use the recommend way to install yarn?

shiftkey committed:
hashtag no parents

nerdneha committed:
time to bump

bump bump bump bump bumnp it up

Co-Authored-By: Brendan Forster <brendan@github.com>

nerdneha committed:
add changelog for version

well actually anyone can bump of a version but only a true releaser will change the changelog

Co-Authored-By: Brendan Forster <brendan@github.com>
